app/helpers/vacancies_options_helper.rb

```diff
@@ -24,6 +23,16 @@ def listed_elsewhere_options
     Vacancy.listed_elsewheres.keys.map { |k| [t("jobs.feedback.listed_elsewhere.#{k}"), k] }
   end
 
+  def mapped_job_location_options(organisation)
+    Vacancy::JOB_LOCATION_OPTIONS.map do |k, _v|
+      next unless MultiSchoolJobsFeature.enabled? || k != :at_multiple_schools
```

Comment:
We should probably get rid of this feature flag, right?

Reply:
If I'm reading it correctly, this should be `&&`, not `||` - what would be the point of rejecting every key-value pair where the key is not `:at_multiple_schools`?

Reply:
Yes I would like to. It would be nice to have some sort of process in place for how long features are live before we remove them, I'll probably raise a PR soon for it, we haven't seen any issues with it anyway and I assume we won't want to turn it off again

Reply:
The next happens only when MultiSchoolJobsFeature is disabled and k == :at_multiple_schools (false || false). Everything else evaluates to true and the code following is executed

Reply:
I'm confused by the number of nested negatives there! ('Unless not...' whu?) But you're right.